db/schema.rb

```diff
@@ -367,6 +367,8 @@
     t.text "dag_vars"
     t.integer "max_input_fragments"
     t.datetime "client_updated_at"
+    t.string "basespace_access_token"
+    t.string "basespace_dataset_id"
```

Comment:
Based on the name "dataset", I would expect there to be a many to many relationship between samples and datasets. If that's not correct, please add a comment somewhere explaining why there is a one to one relationship.

Reply:
No longer storing these in the database and instead passing them into resque as parameters. Added a comment explaining what a dataset is.